src/topp/FeatureFinderCentroided.cpp

```diff
@@ -178,6 +180,9 @@ class TOPPFeatureFinderCentroided :
     PeakFileOptions options;
     options.setMSLevels(vector<Int>(1, 1));
 
+    using RP_TYPE = DRange<1>::PositionType;
+    options.setIntensityRange({std::numeric_limits<RP_TYPE>::epsilon(), RP_TYPE::maxPositive()});
```

Comment:
should be `std::numeric_limits<RP_TYPE>::min` (which is 1.175494351e-38F) not `epsilon` (which is 1.192092896e-07F)

Reply:
note: min is positive. lowest would be -max.

Reply:
yep right will chance